bitwarden_license/src/Portal/Models/PolicyEditModel.cs

```diff
@@ -81,6 +81,8 @@ public Policy ToPolicy(Policy existingPolicy)
                 case PolicyType.PasswordGenerator:
                     existingPolicy.Data = JsonSerializer.Serialize(PasswordGeneratorDataModel, options);
                     break;
+                case PolicyType.OnlyOrg:
+                    break;
```

Comment:
The 2FA policy wasn't represented in this switch at all, which was causing a bug: enabling the 2FA policy from the portal simply didn't work.

I noticed this, but it didn't seem in scope of this PR so I was planning to fast-follow and fix it later, like with deleting orgUser records with an org, but since you brought it up I went ahead and fixed it here 😁